api/BUILD

```diff
@@ -8,11 +7,9 @@ api_proto_library(
     srcs = ["address.proto"],
 )
 
-go_proto_library(
-    name = "address_go_proto",
+api_go_proto_library(
     importpath = "github.com/envoyproxy/data-plane-api/api/address",
-    proto = ":address",
-    visibility = ["//visibility:public"],
+    proto = "address",
     deps = [
         "@com_github_gogo_protobuf//:gogo_proto_go",
         "@com_github_golang_protobuf//ptypes/wrappers:go_default_library",
```

Comment:
Similar to `api_proto_library`, I would bake some of the WKT and validator deps directly into the macro. This makes things less verbose here and I don't think we're really widening the effective dependencies that much.

Reply:
SGTM, let me know if you think any of the ones I've baked in don't belong.